sdk/ml/azure-ai-ml/tests/component/unittests/test_spark_component_entity.py

```diff
@@ -103,3 +103,15 @@ def test_spark_component_version_as_a_function_with_inputs(self):
         rest_yaml_component = yaml_component._to_rest_object()
 
         assert rest_yaml_component == expected_rest_component
+
+    def test_from_rest_object(self):
```

Comment:
this is should be some common test? not just for spark?

Reply:
I have moved it to `test_component_schema.py` and test it with helloworld component (a command component)